### PR TITLE
Update developer profile and UG,DG

### PR DESCRIPTION
**Pull request by yaololo**
Update the following docs:
- developer profile
- UG
- DG

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -330,6 +332,18 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+
+1. When create a contact, user can add a task to that contact
+2. ContactManager shows a list of persons
+3. User is able to see the task that he/she need to do for the contact
+
+**Use case: Edit task to a contact**
+
+1. User requests to list persons
+2. ContactManager shows a list of persons
+3. User is able to edit the contact to update the task to that contact, if no task, user can put `-`
```
Comment (TanChiaChun) [inline docs/DeveloperGuide.md:345]: @yaololo take note of my fix implemented [here](https://github.com/AY2122S2-TIC4002-F18-4/tp2/pull/63)

File: docs/UserGuide.md [hunk 2]
```diff
@@ -115,7 +115,8 @@ Examples:
 * `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
 * `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
 * `edit 1 f/5.55` Edits money owed of the 1st person to be `5.55`.
-* `edit 1 tk/buy birthday cace` Edits the task to `buy birthday cake`.
+* `edit 1 tk/buy birthday cake` Edits the task to `buy birthday cake`.
+* `edit 1 tk/-` Edits the task to `-` which indicate no more task for this person.
```
Comment (TanChiaChun) [inline docs/UserGuide.md:119]: Same as above comment

**Review by TanChiaChun — APPROVED** [review]
LGTM

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (yaololo) [reply docs/DeveloperGuide.md:345]: 👍 thanks! @TanChiaChun

**Pull request merged**